lib/src/shared/main/java/com/couchbase/lite/Document.java

```diff
@@ -26,19 +28,21 @@
 
 import com.couchbase.lite.internal.CBLStatus;
 import com.couchbase.lite.internal.core.C4Constants;
+import com.couchbase.lite.internal.core.C4Database;
 import com.couchbase.lite.internal.core.C4Document;
 import com.couchbase.lite.internal.fleece.FLDict;
 import com.couchbase.lite.internal.fleece.FLEncoder;
 import com.couchbase.lite.internal.fleece.FLSliceResult;
 import com.couchbase.lite.internal.fleece.MRoot;
+import com.couchbase.lite.internal.utils.Preconditions;
 
 
 /**
  * Readonly version of the Document.
  */
 public class Document implements DictionaryInterface, Iterable<String> {
     // !!! This code is from v1.x. Replace with c4rev_getGeneration().
-    static long generationFromRevID(String revID) {
+    private static long generationFromRevID(String revID) {
```

Comment:
is this going to hide one public API?

Reply:
I don't think so.  It was package private before, so only visible to code in the com.couchbase.lite namespace...